source/basic.tex

```diff
@@ -4240,14 +4240,12 @@
 extended signed integer types are collectively called
 \defnx{signed integer types}{signed integer type}.
 \indextext{integral type!implementation-defined \tcode{sizeof}}%
+\begin{note}
 Plain
 \tcode{int}s have the natural size suggested by the architecture of the
-execution environment%
-\footnote{\tcode{int} must also be large enough
-to contain any value in the range
-\crange{INT_MIN}{INT_MAX},
-as defined in the header \tcode{<climits>}.};
+execution environment%;
```

Comment:
The `%` was here to avoid introducing whitespace before the footnote. Since you've removed the footnote, you can remove the `%` as well.